client-common/src/tendermint/types/block_results.rs

```diff
@@ -360,106 +273,59 @@ mod tests {
+            //            let response_str = r#"
+            //            {
+            //    "height":"37",
+            //    "txs_results":[
+            //        {
+            //            "code":0,
+            //            "data":"",
+            //            "log":"",
+            //            "info":"",
+            //            "gasWanted":"0",
+            //            "gasUsed":"0",
+            //            "events":[
+            //                {
+            //                    "type":"valid_txs",
+            //                    "attributes":[
+            //                        {
+            //                            "key":"ZmVl",
+            //                            "value":"MC4wMDAwMDMwNw=="
+            //                        },
+            //                        {
+            //                            "key":"YWNjb3VudA==",
+            //                            "value":"MHgzMzUwMmVkMzlkMGM0ZTIwNDRmYjM3ZmRjZDUxNjE0OTNmNTkwMGMz"
+            //                        },
+            //                        {
+            //                            "key":"dHhpZA==",
+            //                            "value":"ZjFmNzNkNmFjZWMyMTExOGRkMWUzNmY2ODRhYWUyMmM2Y2IxN2ZjNTFhZGEzNGEzNDIzMDlkNTMxY2I5YmU4ZA=="
+            //                        }
+            //                    ]
+            //                }
+            //            ],
+            //            "codespace":""
+            //        }
+            //    ],
+            //    "begin_block_events":null,
+            //    "end_block_events":[
+            //        {
+            //            "type":"block_filter",
+            //            "attributes":[
+            //                {
+            //                    "key":"ZXRoYmxvb20=",
+            //                    "value":"AAAAAAAAAAAAAAAAAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=="
+            //                }
+            //            ]
+            //        }
+            //    ],
+            //    "validator_updates":[],
+            //    "consensus_param_updates":null
+            //}"#;
+            let block_results: BlockResultsResponse =
+                serde_json::from_str(response_str).expect("invalid response str");
             let target_account = StakedStateAddress::from(
-                RedeemAddress::from_str("0xe4a2a719ca933d3f79a8506aa96cefde3405b0a7").unwrap(),
-            );
-            let result = block_results.contains_account(&target_account);
-            assert!(result.is_ok());
-            assert_eq!(true, result.unwrap());
-        }
-
```

Comment:
Any reason to remove the following test cases?

Reply:
yeah, most of the test cases in this file is testing the invalid value of `Attribute` in the `BlockResults` and `BlockResults` defined in this file. 

Now, we use `BlockResultsResponse` directly which defined in `tendermint-rs`, and we query tendermint to get the `BlockResultsResponse`, I don't think tendermint will give us one which value is invalid.